### DISCUSSION TITLE
Students with GitHub Student Developer Pack lost access to it

### DISCUSSION THREAD

**Discussion by OxCone1** [community/community#41379] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Me and my friends that study the same program and are verified students simultaneously lost access to GitHub Copilot.
Now in the subscription tab we are only able to purchase it.

**Comment by Tivikrat** [comment]
1. Go to https://github.com/github-copilot/free_signup (auto-redirect to https://github.com/settings/billing is acceptable).
2. If https://github.com/settings/billing shows you "You are eligible to use GitHub Copilot for free.", then retry login at your IDE plugin.

**Comment by OxCone1** [comment, marked as answer]
Ok, the issue seems to be gone now (11.12.2022 0:07 UTC +0)

**Comment by haakon-e** [comment]
It stopped working for a hot second, and now it seems to be back online for me. Try logging out/in of GitHub in VSCode, restarting computer, looking for VSCode updates, etc.

**Comment by OxCone1** [comment]
Lemme tell you guys, that thing got me scared af

**Comment by Akash1134** [comment]
👋 Hi there, Thank you everyone for taking the time to report this issue.

Apologies for that temporary issue on our end. Please retry logging in to your IDE plugin, and Copilot should work as expected. 

If you run into any further issues, please reply back in detail with any useful screenshots.

**Reply by harryisfish** [reply]
<img width="372" alt="Screenshot 2022-12-14 at 16 05 50" src="https://user-images.githubusercontent.com/15265627/207540064-b9ff8951-5733-436e-85fb-88df4770f2c1.png">
Hi, I am a verified student. And I got the email which told me that my free GitHub copilot access has expired this morning. I tried logging in to VSCode plugin and it does not work.

**Reply by NotaGames** [reply]
I just got this same issue today.

**Reply by Qiming-Liu** [reply]
Same here

**Reply by jz0ojiang** [reply]
Same issue here

**Reply by perezromeojohn** [reply]
Same here

**Reply by sena-nana** [reply]
Same here

**Reply by sena-nana** [reply]
![image](https://user-images.githubusercontent.com/34237511/211209323-2db0aac6-fc1f-4ff5-9e47-a0e8741db003.png)

**Reply by yunmango** [reply]
me too

**Reply by sena-nana** [reply]
I found this.Maybe this error will help you to check the reasons.
![image](https://user-images.githubusercontent.com/34237511/211209552-913eabd4-c409-40e7-93c4-8e85e064cf06.png)

**Reply by acarmonag** [reply]
same

**Comment by HikaruChang** [comment]
I had the same problem. It stopped working just now. But my student certification has not expired.

![DB675F5D-ACB5-4F43-8653-BC6D83623935](https://user-images.githubusercontent.com/16264281/211208919-400685e8-bd11-4767-be2a-d1567c9520aa.png)
![E4977FB2-8967-4A7F-BCDC-017B1C9F4C4C](https://user-images.githubusercontent.com/16264281/211208914-d760cc82-7b18-4175-966c-ec3f4d389373.png)

**Reply by Ninja-Yubaraj** [reply]
mine too stopped working!

**Reply by NawrasBukhari** [reply]
have you managed to solve it?

**Reply by AhmetUsluoglu** [reply]
same problem here.

**Reply by HikaruChang** [reply]
I received an email, which has only been renewed for one month

![1B6A4AEF-55B5-4945-8B76-C3696F2A3EF0](https://user-images.githubusercontent.com/16264281/211234121-22c27fd8-81e8-487e-a44f-ccd2c14e3491.jpeg)
![3751589B-B204-44C4-8EC3-C43AA71D67E3](https://user-images.githubusercontent.com/16264281/211234124-002708e1-89ee-4a83-8eee-ee6c51617041.jpeg)

**Reply by HikaruChang** [reply]
I seem to have found the answer.
![17D26F6C-8531-4D4E-B689-3B48B323FF04](https://user-images.githubusercontent.com/16264281/211235513-57d393d7-064c-4986-b48c-c9bed1e2629f.jpeg)

**Reply by JinyiW** [reply]
same problem here.

**Comment by Ninja-Yubaraj** [comment]
![image](https://user-images.githubusercontent.com/41476315/211216012-8837b662-3f5b-4154-a0f5-f6297657390b.png)
Got the same issue.

**Comment by NawrasBukhari** [comment]
This issue still exists 01/09/2023

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by otkrickey** [comment]
Solved: #43634

**Comment by IlkerDEMIR-s** [comment]
Even though I'm a verified student the copilot has stopped working and now it's asking for a premium membership. Please help me ?


![Untitled](https://user-images.githubusercontent.com/115078996/211540730-b39a33f9-ed25-4acd-9b06-1ad910028a52.png)

******************************************************************************

EDİT:

I just got this email from Github then Copilot was renewed and started working. :D

![image](https://user-images.githubusercontent.com/115078996/211543397-7315e788-0cbe-47c1-b7b6-65dfcbaec6b6.png)

**Comment by erbanku** [comment]
Same here

![https://github.zendesk.com/attachments/token/3BBvH23ljpZD7iIEZLVeIBrza/?name=image.png](https://github.zendesk.com/attachments/token/3BBvH23ljpZD7iIEZLVeIBrza/?name=image.png)
![https://github.zendesk.com/attachments/token/AW87sL2CDFJNVpzh25N6tonq6/?name=image.png](https://github.zendesk.com/attachments/token/AW87sL2CDFJNVpzh25N6tonq6/?name=image.png)
